### ISSUE TITLE
Doesn't work with milkdown v7

### ISSUE THREAD

**Issue by knownasilya** [HexMox/milkdown-plugin-placeholder#3]
Throws an error

```
index.es.js:14 Uncaught (in promise) TypeError: t.create is not a function
    at P.eval [as inject] (index.es.js:14:1781)
    at placeholder (index.js:12:456)
 
index.es.js:25 Uncaught (in promise) MilkdownError: Context "s" not found, do you forget to inject it?
    at f (index.es.js:25:1333)
    at b.eval [as get] (index.es.js:14:810)
    at P.eval [as use] (index.es.js:14:2100)
    at P.eval [as set] (index.es.js:14:2168)
```

**Comment by HexMox** [comment]
The nord theme of playground is broken after upgrade to milkdown v7 and I have pushed my code in main branch.

PR is welcome!

**Comment by knownasilya** [comment]
I think nord goes on `editor.config(nord)` now? Instead of `use`

**Comment by HexMox** [comment]
> I think nord goes on `editor.config(nord)` now? Instead of `use`

The style of `<code>`, `<img>` etc. is still broken though I setting up [tailwindcss](https://tailwindcss.com/) and some plugins and  according to https://milkdown.dev/docs/api/theme-nord. But the placeholder seems to work correctly.

I have released v0.2.0 for milkdown v7

**Comment by knownasilya** [comment]
Seeing this with 0.2.0

```
index.es.js:24 TypeError: Cannot read properties of undefined (reading 'waitTimers')
    at eval (index.js:23:19)
    at Generator.next (<anonymous>)
    at eval (tslib.es6.js:93:71)
    at new Promise (<anonymous>)
    at __awaiter (tslib.es6.js:89:12)
    at eval (index.js:22:69)
    at eval (index.es.js:350:23)
    at Array.map (<anonymous>)
    at Ce.eval (index.es.js:346:44)
    at Ce.create (index.es.js:368:20)
```

**Comment by ivanminutillo** [comment]
> Seeing this with 0.2.0
> 
> ```
> index.es.js:24 TypeError: Cannot read properties of undefined (reading 'waitTimers')
>     at eval (index.js:23:19)
>     at Generator.next (<anonymous>)
>     at eval (tslib.es6.js:93:71)
>     at new Promise (<anonymous>)
>     at __awaiter (tslib.es6.js:89:12)
>     at eval (index.js:22:69)
>     at eval (index.es.js:350:23)
>     at Array.map (<anonymous>)
>     at Ce.eval (index.es.js:346:44)
>     at Ce.create (index.es.js:368:20)
> ```

fyi i've got the same issue above, using with milkdown v7

**Comment by HexMox** [comment]
> Seeing this with 0.2.0
> 
> ```
> index.es.js:24 TypeError: Cannot read properties of undefined (reading 'waitTimers')
>     at eval (index.js:23:19)
>     at Generator.next (<anonymous>)
>     at eval (tslib.es6.js:93:71)
>     at new Promise (<anonymous>)
>     at __awaiter (tslib.es6.js:89:12)
>     at eval (index.js:22:69)
>     at eval (index.es.js:350:23)
>     at Array.map (<anonymous>)
>     at Ce.eval (index.es.js:346:44)
>     at Ce.create (index.es.js:368:20)
> ```

 Incorrect build of 0.2.0. I have released 0.2.1 and tested in https://stackblitz.com/edit/github-7xktcp?file=src%2Fmain.ts

**Comment by knownasilya** [comment]
Nice, it works
